pegasus/emails/hoc_signup_2017_receipt.md

```diff
@@ -0,0 +1,121 @@
+---
```

Comment:
A markdown file shouldn't have this logic. It belongs in the `HocSignup2017` class.